### ISSUE TITLE
Jupyterhub installation steps do not work locally

### ISSUE THREAD

**Issue by adivekar-utexas** [jupyterhub/jupyterhub#3744]
### Bug description

The basic [installation-and-test steps](https://github.com/jupyterhub/jupyterhub#installation) do not work on a local machine (i.e. using localhost). This is rather counterintuitive...the first thing I do when installing a new Python package is try to run it locally, not run it on a Kubernetes cluster or some other distributed setup. 

#### Expected behaviour

Would request to either: 
1. Update the [documentation](https://github.com/jupyterhub/jupyterhub#installation) to add steps for a minimal local installation using localhost. [These steps](https://stackoverflow.com/a/51185134) were the simplest working example which I found after ~1 hour of Googling.
2. Make the default "jupyterhub" command run in localhost most. 

#### Actual behaviour

The `jupyterhub` command starts a proxy and a server, which is not accessible on a local laptop using the URL provided in the instructions ("Visit `https://localhost:8000`")

### How to reproduce

Follow the Installation steps in the main Github link.

### Your personal set up

I used `conda 4.7.12` and the conda installation steps to setup. 

 - OS:
 

Mac OSX Big Sur version 11.6.2 (20G314)
 - Version(s):
 `conda 4.7.12` with `python=3.8`
 

- <details><summary>Full environment</summary>

Output of `conda list`
```
# Name                    Version                   Build  Channel
alembic                   1.7.5              pyhd8ed1ab_0    conda-forge
anyio                     3.4.0            py38h50d1736_0    conda-forge
appnope                   0.1.2            py38h50d1736_2    conda-forge
argon2-cffi               21.3.0             pyhd8ed1ab_0    conda-forge
argon2-cffi-bindings      21.2.0           py38h96a0964_1    conda-forge
async_generator           1.10                       py_0    conda-forge
attrs                     21.4.0             pyhd8ed1ab_0    conda-forge
babel                     2.9.1              pyh44b312d_0    conda-forge
backcall                  0.2.0              pyh9f0ad1d_0    conda-forge
backports                 1.0                        py_2    conda-forge
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
bleach                    4.1.0              pyhd8ed1ab_0    conda-forge
blinker                   1.4                        py_1    conda-forge
brotlipy                  0.7.0           py38h96a0964_1003    conda-forge
c-ares                    1.18.1               h0d85af4_0    conda-forge
ca-certificates           2021.10.8            h033912b_0    conda-forge
certifi                   2021.10.8        py38h50d1736_1    conda-forge
certipy                   0.1.3                      py_0    conda-forge
cffi                      1.15.0           py38h1a44b6c_0    conda-forge
charset-normalizer        2.0.9              pyhd8ed1ab_0    conda-forge
configurable-http-proxy   4.5.0           node17_hda8ccec_2    conda-forge
cryptography              36.0.1           py38h56c4533_0    conda-forge
debugpy                   1.5.1            py38ha048514_0    conda-forge
decorator                 5.1.0              pyhd8ed1ab_0    conda-forge
defusedxml                0.7.1              pyhd8ed1ab_0    conda-forge
entrypoints               0.3             pyhd8ed1ab_1003    conda-forge
flit-core                 3.6.0              pyhd8ed1ab_0    conda-forge
greenlet                  1.1.2            py38ha048514_1    conda-forge
icu                       69.1                 he49afe7_0    conda-forge
idna                      3.1                pyhd3deb0d_0    conda-forge
importlib-metadata        4.10.0           py38h50d1736_0    conda-forge
importlib_resources       5.4.0              pyhd8ed1ab_0    conda-forge
ipykernel                 6.6.0            py38h5fd9f69_0    conda-forge
ipython                   7.30.1           py38h50d1736_0    conda-forge
ipython_genutils          0.2.0                      py_1    conda-forge
jedi                      0.18.1           py38h50d1736_0    conda-forge
jinja2                    3.0.3              pyhd8ed1ab_0    conda-forge
json5                     0.9.5              pyh9f0ad1d_0    conda-forge
jsonschema                4.3.2              pyhd8ed1ab_0    conda-forge
jupyter_client            7.1.0              pyhd8ed1ab_0    conda-forge
jupyter_core              4.9.1            py38h50d1736_1    conda-forge
jupyter_server            1.13.1             pyhd8ed1ab_0    conda-forge
jupyter_telemetry         0.1.0              pyhd8ed1ab_1    conda-forge
jupyterhub                2.0.1                hd8ed1ab_0    conda-forge
jupyterhub-base           2.0.1              pyhd8ed1ab_0    conda-forge
jupyterlab                3.2.5              pyhd8ed1ab_0    conda-forge
jupyterlab_pygments       0.1.2              pyh9f0ad1d_0    conda-forge
jupyterlab_server         2.10.2             pyhd8ed1ab_0    conda-forge
krb5                      1.19.2               hcfbf3a7_3    conda-forge
libcurl                   7.80.0               hf45b732_1    conda-forge
libcxx                    12.0.1               habf9029_1    conda-forge
libedit                   3.1.20191231         h0678c8f_2    conda-forge
libev                     4.33                 haf1e3a3_1    conda-forge
libffi                    3.4.2                h0d85af4_5    conda-forge
libnghttp2                1.43.0               h6f36284_1    conda-forge
libsodium                 1.0.18               hbcb3906_1    conda-forge
libssh2                   1.10.0               h52ee1ee_2    conda-forge
libuv                     1.42.0               h0d85af4_0    conda-forge
libzlib                   1.2.11            h9173be1_1013    conda-forge
mako                      1.1.6              pyhd8ed1ab_0    conda-forge
markupsafe                2.0.1            py38h96a0964_1    conda-forge
matplotlib-inline         0.1.3              pyhd8ed1ab_0    conda-forge
mistune                   0.8.4           py38h96a0964_1005    conda-forge
nbclassic                 0.3.4              pyhd8ed1ab_0    conda-forge
nbclient                  0.5.9              pyhd8ed1ab_0    conda-forge
nbconvert                 6.3.0            py38h50d1736_1    conda-forge
nbformat                  5.1.3              pyhd8ed1ab_0    conda-forge
ncurses                   6.2                  h2e338ed_4    conda-forge
nest-asyncio              1.5.4              pyhd8ed1ab_0    conda-forge
nodejs                    17.1.0               h0da8292_2    conda-forge
notebook                  6.4.6              pyha770c72_0    conda-forge
oauthlib                  3.1.1              pyhd8ed1ab_0    conda-forge
openssl                   1.1.1l               h0d85af4_0    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pamela                    1.0.0                      py_0    conda-forge
pandoc                    2.16.2               h0d85af4_0    conda-forge
pandocfilters             1.5.0              pyhd8ed1ab_0    conda-forge
parso                     0.8.3              pyhd8ed1ab_0    conda-forge
pexpect                   4.8.0              pyh9f0ad1d_2    conda-forge
pickleshare               0.7.5                   py_1003    conda-forge
pip                       21.3.1             pyhd8ed1ab_0    conda-forge
prometheus_client         0.12.0             pyhd8ed1ab_0    conda-forge
prompt-toolkit            3.0.24             pyha770c72_0    conda-forge
psutil                    5.9.0            py38h96a0964_0    conda-forge
ptyprocess                0.7.0              pyhd3deb0d_0    conda-forge
pycparser                 2.21               pyhd8ed1ab_0    conda-forge
pycurl                    7.44.1           py38h51ed575_1    conda-forge
pygments                  2.11.1             pyhd8ed1ab_0    conda-forge
pyjwt                     2.3.0              pyhd8ed1ab_1    conda-forge
pyopenssl                 21.0.0             pyhd8ed1ab_0    conda-forge
pyparsing                 3.0.6              pyhd8ed1ab_0    conda-forge
pyrsistent                0.18.0           py38h96a0964_0    conda-forge
pysocks                   1.7.1            py38h50d1736_4    conda-forge
python                    3.8.12          h17280f6_2_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python-json-logger        2.0.1              pyh9f0ad1d_0    conda-forge
python_abi                3.8                      2_cp38    conda-forge
pytz                      2021.3             pyhd8ed1ab_0    conda-forge
pyzmq                     22.3.0           py38hd3b92b6_1    conda-forge
readline                  8.1                  h05e3726_0    conda-forge
requests                  2.26.0             pyhd8ed1ab_1    conda-forge
ruamel.yaml               0.17.19          py38h96a0964_0    conda-forge
ruamel.yaml.clib          0.2.6            py38h96a0964_0    conda-forge
send2trash                1.8.0              pyhd8ed1ab_0    conda-forge
setuptools                60.2.0           py38h50d1736_0    conda-forge
six                       1.16.0             pyh6c4a22f_0    conda-forge
sniffio                   1.2.0            py38h50d1736_2    conda-forge
sqlalchemy                1.4.29           py38h96a0964_0    conda-forge
sqlite                    3.37.0               h23a322b_0    conda-forge
terminado                 0.12.1           py38h50d1736_1    conda-forge
testpath                  0.5.0              pyhd8ed1ab_0    conda-forge
tk                        8.6.11               h5dbffcc_1    conda-forge
tornado                   6.1              py38h96a0964_2    conda-forge
traitlets                 5.1.1              pyhd8ed1ab_0    conda-forge
urllib3                   1.26.7             pyhd8ed1ab_0    conda-forge
wcwidth                   0.2.5              pyh9f0ad1d_2    conda-forge
webencodings              0.5.1                      py_1    conda-forge
websocket-client          1.2.3              pyhd8ed1ab_0    conda-forge
wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
xz                        5.2.5                haf1e3a3_1    conda-forge
zeromq                    4.3.4                he49afe7_1    conda-forge
zipp                      3.6.0              pyhd8ed1ab_0    conda-forge
zlib                      1.2.11            h9173be1_1013    conda-forge
```
</details>

- <details><summary>Configuration</summary>

```python
N/A the default was used
```
</details>

- <details><summary>Logs</summary>

```
N/A
```
</details>

**Comment by minrk** [comment]
There's a typo that it should be `http://localhost:8000`, not `https://localhost:8000`, but all of the actual install steps in the README produces a working localhost install on a current mac, assuming you have Python and/or conda.

For conda, this is all you need to get running (stitching together the commands without the explanation):

```python
conda install jupyterhub
conda install jupyerlab # for singleuser-servers
jupyterhub
# visit http://localhost:8000/
```

[sample installation recording](https://asciinema.org/a/459640).

**Comment by minrk** [comment]
The typo is fixed in #3747
